src/niscope/metadata/functions_addon.py

```diff
@@ -696,6 +698,39 @@
             'description': 'Retrieves the custom coefficients for the equalization FIR filter on the device. This filter is designed to compensate the input signal for artifacts introduced to the signal outside of the digitizer. Because this filter is a generic FIR filter, any coefficients are valid. Coefficient values should be between +1 and –1.',
         },
     },
+    # niScope metadata is missing error_message but we need it for error handling
```

Comment:
Please add the NI internal bug number here, so we know when it's safe to remove.